content/en/docs/reference/config-api/kubeadm-config.v1beta2.md

```diff
@@ -712,7 +712,7 @@ for, so other administrators can know its purpose.</td>
     
   
 <tr><td><code>ttl</code> <B>[Required]</B><br/>
-<code>invalid type</code>
+<a href="https://godoc.org/k8s.io/apimachinery/pkg/apis/meta/v1#Duration"><code>meta/v1.Duration</code></a>
```

Comment:
for some reason metav1.Duration is not present at https://kubernetes.io/docs/reference/generated/kubernetes-api/

Reply:
Well ... let me try find out where `Duration` is documented in the reference. I always have a problem finding things in the new version of the API reference.

Reply:
@neolit123 I failed to find generated reference for meta data types. It is not in the old API reference, and it is not in the new version. That is something for the API reference generator to improve.

As a side note, I'd suggest kubeadm to use `ttlSeconds` rather than `ttl`. `ttlSeconds` can be easily understood by human and all programming languages. Duration, ..., is weird.

Reply:
> I failed to find generated reference for meta data types. It is not in the old API reference, and it is not in the new version. That is something for the API reference generator to improve.

could be due to the fact that the Duration struct is defined in a duration.go file in the same v1 package (not in types.go).

> As a side note, I'd suggest kubeadm to use ttlSeconds rather than ttl. ttlSeconds can be easily understood by human and all programming languages. Duration, ..., is weird.

for the next version v1beta4 i was going to propose to use `*Seconds` (`int` types) everywhere instead of metav1.Duration.
metav1.Duration is not CRD friendly.